docs/articles/documentation/test-api/actions/action-options.md

```diff
@@ -6,12 +6,48 @@ checked: true
 ---
 # Action Options
 
-This topic describes three types of [test action](index.md) options.
+This topic describes [test action](index.md) options.
 
+* [Basic Action Options](#basic-action-options)
 * [Mouse Action Options](#mouse-action-options)
 * [Click Action Options](#click-action-options)
 * [Typing Action Options](#typing-action-options)
 
+## Basic Action Options
```

Comment:
It's correct for internal code structure, but as an user i think basic options, it are options, which work for all actions.